INTITA/Modules/Profile/ProfileViewController.swift

```diff
@@ -7,19 +7,22 @@
 
 import UIKit
 
-class ProfileViewController: UITableViewController, Storyboarded {
-    private let rowNumber = 6
+class ProfileViewController: UIViewController, Storyboarded {
+    private let rowNumber = 5
     private let alert: AlertView = .fromNib()
     
     weak var coordinator: ProfileCoordinator?
     var viewModel: ProfileViewModel?
     
+    @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var headerView: UIView!
+    lazy var headerViewContent: ProfileHeaderViewCell = .fromNib()
+    
     override var preferredStatusBarStyle: UIStatusBarStyle { .lightContent }
     
-    //MARK: - View life cycle
+    //MARK: - View lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
-        colorStatusBar(UIColor.primaryColor)
```

Comment:
This one doesn't conflict with the white screen in the bottom below the footer now?